zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/AzureNotebookRepo.java

```diff
@@ -205,8 +205,21 @@ public void close() {
   }
 
   @Override
-  public void checkpoint(String noteId, String checkPointName) throws IOException {
+  public Rev checkpoint(String noteId, String checkpointMsg) throws IOException {
```

Comment:
good point, so I was expecting to revise and improve naming during PR review. Actually originally it was as `Rev` in the `NotebookRepoVersioned` class, and although I modified the class itself but name wasn't changed. I agree that having full name as `Revision` for that class makes more sense :+1: